sktime/utils/plotting.py

```diff
@@ -64,13 +62,21 @@ def plot_series(*series, labels=None, markers=None):
     else:
         markers = ["o" for _ in range(n_series)]
 
+    def _type(id):
+        return str(type(id)).split(".")[-1].split("'")[0]
+
     # create combined index
     index = series[0].index
+    idx_type = type(index)
     for y in series[1:]:
         # check types, note that isinstance() does not work here because index
         # types inherit from each other, hence we check for type equality
+        acc_dtypes = ["RangeIndex", "Int64Index", "UInt64Index", "Float64Index"]
```

Comment:
Note that we already check if the index is valid above in `check_y(y)`.

Reply:
Yep, I just went through the code for this function and it seems like the only accepted index types as of now are Int64Index, RangeIndex, PeriodIndex and pd.DatetimeIndex. Can we add Float64Index and UInt64Index in this list too?

Reply:
Float64Index and UInt64Index index types don't really make sense for time indices I think, at least not for time series with equidistant time points. Why would you want to add them?

Reply:
I was thinking more along the lines of how the function shouldn't throw an error if the user passes a series with index values as '1.0, 2.0, 3.0, and so on.' But yeah, otherwise, including them doesn't really make sense, so we could leave them off

Reply:
Ah okay, I see, but I think it's probably safer to tell people to use an integer index in that case rather than trying to convert a float index.